tests/kafka/server.properties-with-ssl-sasl.diff

```diff
@@ -0,0 +1,34 @@
+*** server.properties.orig	2025-05-12 14:13:30.855032598 +0200
```

Comment:
The diffs should be moved to the first patch, because that's where we explain how to set these properties.